metrics/prometheus/prometheus.go

```diff
@@ -0,0 +1,28 @@
+package prometheus
+
+import "github.com/docker/go-metrics"
+
+const (
+	// Namespace is the namespace of prometheus metrics
+	Namespace = "docker_distribution"
+)
+
+var (
+	// HealthNamespace is the prometheus namespace of health endpoint
+	HealthNamespace = metrics.NewNamespace(Namespace, "http", metrics.Labels{"handler": "health"})
+
+	// storageNamespace is the prometheus namespace of blob/cache related operations
+	storageNamespace = metrics.NewNamespace(Namespace, "storage", nil)
+)
+
+var (
+	// BlobAction is the metrics of blob related operations
+	BlobAction = storageNamespace.NewLabeledTimer("blob_action", "The number of seconds it takes to get/put blob", "action")
+
+	// CacheRequestCount is the number of total cache request received/hits/misses
+	CacheRequestCount = storageNamespace.NewLabeledCounter("cache_request", "The number of cache request received", "type")
```

Comment:
`Cache` or something like that is fine. `CacheRequest` makes me think of counting requests that were cached.